.github/workflows/ci-pr.yml

```diff
@@ -141,9 +141,9 @@ jobs:
           - setup: linux-x86_64-java11
             docker-compose-build: "-f docker/docker-compose.yaml -f docker/docker-compose.centos-6.111.yaml build"
             docker-compose-run: "-f docker/docker-compose.yaml -f docker/docker-compose.centos-6.111.yaml run build-leak"
-          - setup: linux-x86_64-java11-graal
-            docker-compose-build: "-f docker/docker-compose.yaml -f docker/docker-compose.centos-6.graalvm111.yaml build"
-            docker-compose-run: "-f docker/docker-compose.yaml -f docker/docker-compose.centos-6.graalvm111.yaml run build-leak"
+          - setup: linux-x86_64-java17-graal
+            docker-compose-build: "-f docker/docker-compose.yaml -f docker/docker-compose.centos-6.graalvm117.yaml build"
+            docker-compose-run: "-f docker/docker-compose.yaml -f docker/docker-compose.centos-6.graalvm117.yaml run build-leak"
           - setup: linux-x86_64-java17
             docker-compose-build: "-f docker/docker-compose.yaml -f docker/docker-compose.centos-7.117.yaml build"
```

Comment:
A bit unrelated to the PR, is the intention of this build to run the tests on Centos 7 using Java 17?
If yes, I may be wrong but it seems like these may be running on Centos 6:
```
Run docker-compose -f docker/docker-compose.yaml -f docker/docker-compose.centos-7.117.yaml build
The GPG_PRIVATE_KEY variable is not set. Defaulting to a blank string.
The GPG_PASSPHRASE variable is not set. Defaulting to a blank string.
The GPG_KEYNAME variable is not set. Defaulting to a blank string.
stage-snapshot uses an image, skipping
build uses an image, skipping
build-boringssl-static uses an image, skipping
build-leak-boringssl-static uses an image, skipping
deploy uses an image, skipping
build-leak uses an image, skipping
shell uses an image, skipping
build-unsafe-buffer uses an image, skipping
stage-release uses an image, skipping
common uses an image, skipping
Building runtime-setup
Sending build context to Docker daemon  17.41kB
Step 1/11 : FROM centos:6.10
```

Reply:
Yeah, I think that was the intention, and yeah it looks wrong. Not something to hold back this PR, though.

Reply:
Sounds good - should I open an issue for this to keep it tracked?